tools/factories.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import path from 'path';
+import fsp from 'fs-promise';
+import { exec } from 'child-process-promise';
```

Comment:
Can you use exec from tools/exec? That one is configured to handle verbose output if requested.